### PR TITLE
Parcial01 - Added features in description

### PR DESCRIPTION
**Pull request by basaar24**
Added member-edit component
Updated member-edit component
Added NgForm and @ViewChild
Added can deactivate guard
Added API support to persist user edition
Updating the user in the SPA app
Added loading indicator/service
Storing the state of the app via service

### CODE REVIEW

**Comment by AntoMB3** [comment]
Problemas en ticket #37632376

File: API/API.csproj [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="12.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.0">
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.1" />
```
Comment (AntoMB3) [inline API/API.csproj:11]: Checar versión actualizada

**Review by AntoMB3 — COMMENTED** [review]
Checar revisión pendiente

File: API/API.csproj (hunk 1 above)
Comment (basaar24) [reply API/API.csproj:11]: Hola,

Ya he aplicado sus comentarios.

¡Gracias!

**Review by AntoMB3 — APPROVED** [review]
Aprobado

**Pull request merged**